resolwe_bio/docker_images/chipseq/packages-manual/r-packages.sh

```diff
@@ -5,5 +5,12 @@
 # XXX: This is unverifiable and thus may compromise the whole image.
 # XXX: Use notary (https://github.com/ropenscilabs/notary) when ready.
 Rscript --slave --no-save --no-restore-history -e " \
-  install.packages('spp', dependencies=TRUE) \
+  install.packages(c('spp', 'devtools'), dependencies=TRUE) \
 "
+
+# XXX: This is unverifiable and thus may compromise the whole image.
+# XXX: Use notary (https://github.com/ropenscilabs/notary) when ready.
+
+Rscript --slave --no-save --no-restore-history -e " \
+  devtools::install_github('genialis/ChIPQC') \
```

Comment:
ChIP-seq docker image has a readme, where you can list additional package that is present in the image.